enterprise-suite/Makefile

```diff
@@ -90,53 +83,13 @@ lint-allyaml: allyaml  ## Check that all.yaml is valid (Need Minikube running)
 clean::  ## Delete make artifacts
 	rm -f $(HELM_CHARTS_DIR)/docs/$(RELEASE_NAME)/$(ALL_YAML)
 
-
-## These targets used for dev work
-#
-
-# Expose services using NodePorts.  Normally disabled but often used
-# with minikube.  We enable for debugging purposes.
-# (If desired, you can override with something like "make <target> EXPOSE_METHOD=LoadBalancer"
-# or "make <target> EXPOSE_METHOD=" to get the default unexposed behavior.)
-EXPOSE_SERVICES=--set exposeServices=NodePort
```

Comment:
It looks like EXPOSE_SERVICES is undefined now, so what of the reference on line 115/91 below?  That either needs to be removed (in which case we can only use the default), or we need some value defined.

Reply:
Good point, I missed that. But I'm not sure we should be exposing nodeport by default in all.yml.

Reply:
Yeah I didn't want to change it. But we should just remove all.yaml as I keep saying :).